services/app-web/src/testHelpers/apolloHelpers.tsx

```diff
@@ -51,15 +54,15 @@ function mockValidCMSUser(): UserType {
     }
 }
 
-export function mockDraft(): DraftSubmission {
+export function mockDraft(): UnlockedHealthPlanFormDataType {
```

Comment:
These mock form datas could be moved into the domain-mocks package so we can start to consolidate them into one place.